### PR TITLE
Setup dev. env. with docker

### PR DESCRIPTION
**Pull request by thamulot**
Working with my Mac I had a hard time trying to setup myself to begin the development of https://github.com/socram8888/tonyhax/issues/75

I followed the `Development` intructions at https://orca.pet/tonyhax/ as well as the `Compiling` instructions from https://github.com/Lameguy64/mkpsxiso#compiling.

I got an error with the `gcc-10-mips-linux-gnu` package not providing the `mips-linux-gnu-gcc` CLI which I fixed (and you should probably too in the aformentioned doc) with installing `gcc-mips-linux-gnu` instead.

## Usage

Basically you clone the tonyhax repo. Then once inside the dir you have to build the docker image with,

```sh
docker build -t tonyhax-dev .
```

Now that you have the image, you can compile tonyhax by booting a container,

```sh
docker run -v $(pwd):/tonyhax -it tonyhax-dev
```

And then `make`,

```sh
cd tonyhax/
make
```

The zip file will appear at the root of the host directory (outside the container) 🎉

### CODE REVIEW

**Comment by alex-free** [comment]
IMO the better way is to just integrate cross tool-by directly to make any OS capable of building the tool chain.

**Comment by socram8888** [comment]
I fixed the docs. Thanks for the tip.
